src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -33,6 +33,10 @@
           - if @bs_request.superseded_by.present?
             by
             = link_to(@bs_request.superseded_by, number: @bs_request.superseded_by)
+
+        - if @bs_request.accept_at.present?
+          %span.badge.badge-warning.ml-1 auto-accept
```

Comment:
Following up on my previous comment, this line will become:

```suggestion
          %span.badge.alert-warning.ml-1 auto-accept
```

Reply:
updated!